package.json

```diff
@@ -88,5 +90,16 @@
   },
   "files": [
     "dist"
-  ]
-}
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npm test"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
+      "eslint --fix"
```

Comment:
If we do decide to keep this why not just change this to `npm run lint:fix` to keep it all in one place?